specs/gloas/beacon-chain.md

```diff
@@ -252,8 +254,34 @@ class SignedExecutionPayloadEnvelope(Container):
     signature: BLSSignature
 ```
 
+#### `PayloadStatus`
+
+```python
+class PayloadStatus(enum.Enum):
+    EMPTY = 0
+    FULL = 1
+```
+
 ### Modified containers
 
+#### `AttestationData`
+
+*Note*: The `AttestationData` container is modified to contain a
+`PayloadStatus`. The `Attestation`, `SingleAttestation` and `IndexedAttestation` containers are modified indirectly.
+The field `index` is removed from the `AttestationData` and a new field `payload_status` is added.
+
+```python
+class AttestationData(Container):
+    slot: Slot
+    # [Modified in Gloas:EIP7732]
+    # Removed index
+    beacon_block_root: Root
+    # [New in Gloas:EIP7732]
+    payload_status: PayloadStatus
```

Comment:
Since `PayloadStatus` is not an SSZ type, I think it would be better to rename the field and use a boolean.

Maybe `payload_present` or `payload_revealed` (not sure which is better):

```suggestion
    payload_revealed: boolean
```